### DISCUSSION TITLE
Feature Request setting for SetOption24 pressure units inHg

### DISCUSSION THREAD

**Discussion by TheNutHouse84** [arendst/Tasmota#15350] [Ideas]
Request to add support for native display/output of inHg for USA. Currently only support for hPa and mmHg is available.

**Comment by barbudor** [comment]
Hi
It's not obvious to implement it because SetOption24 is part of a binary option group (can only have 0 or 1 as values).
Not sure if it is feasible without a breaking change.
Still thinking about it...

**Reply by vic42** [reply]
What about another binary SetOption that sets "Imperial Units" to 1 or 0? ;-)

**Comment by barbudor** [comment]
I'm not sure switching all units at the same time would be good.
I think UK use imperial for distance but C for temperature.
Theo said he's looking at something more generic. Let's wait and see

**Reply by sfromis** [reply]
Arguably, technically Fahrenheit was not even an "imperial" unit, as that was a number of measures for trade. And temperature is not sold. With the big mess of weirdo "imperial" units, that is a big set of units, not something coherent for making *one* selection.
